packages/plugin-commands-setup/src/setup.ts

```diff
@@ -84,28 +84,34 @@ async function updateShell (currentShell: string | null, pnpmHomeDir: string): P
 }
 
 async function setupShell (configFile: string, pnpmHomeDir: string): Promise<string> {
-  if (!fs.existsSync(configFile)) return `Could not setup pnpm. No ${configFile} found`
+  const content = `export PNPM_HOME="${pnpmHomeDir}"
+export PATH="$PNPM_HOME:$PATH"
+`
+  if (!fs.existsSync(configFile)) {
+    await fs.promises.writeFile(configFile, content, 'utf8')
+    return `Created ${configFile}`
+  }
   const configContent = await fs.promises.readFile(configFile, 'utf8')
   if (configContent.includes('PNPM_HOME')) {
     return `PNPM_HOME is already in ${configFile}`
   }
-  await fs.promises.writeFile(configFile, `${configContent}
-export PNPM_HOME="${pnpmHomeDir}"
-export PATH="$PNPM_HOME:$PATH"
-`, 'utf8')
+  await fs.promises.appendFile(configFile, `\n${content}`, 'utf8')
```

Comment:
In the case of inserting content to rc files. Do you think if it's better to introduce `pnpm setup block`( i.e. two comment lines)?

For instance:

```
# >>> pnpm setup >>>
export PNPM_HOME="${pnpmHomeDir}"
export PATH="$PNPM_HOME:$PATH"
# <<< pnpm setup <<<
```

Next time, if we need update those content insert by `pnpm setup`, it might be easier to find legacy content.

Reply:
sounds good

Reply:
Done